examples/ip64-router/ip64-router.c

```diff
@@ -0,0 +1,27 @@
+#include "contiki.h"
+#include "contiki-net.h"
+#include "ip64.h"
```

Comment:
The platform should have a `MODULE += core/net/ip64` and with that, it *should* be enough to include only `"ip64.h"`. What platform are you compiling for?